src/index.js

```diff
@@ -1,7 +1,309 @@
+      countyLayer: L.geoJSON(CountyData),
+      modalOpen: false,
+      countyGuess: 'Addison',
+      correctGuess: false,
+      score: 100
+    };
+    // bind functions
+    this.moveMarker = this.moveMarker.bind(this);
+    this.clickStart = this.clickStart.bind(this);
+    this.startGame = this.startGame.bind(this);
+    this.handleGiveup = this.handleGiveup.bind(this);
+    this.openModal = this.openModal.bind(this);
+    this.afterOpenModal = this.afterOpenModal.bind(this);
+    this.closeModal = this.closeModal.bind(this);
+    this.handleGuess= this.handleGuess.bind(this);
+    this.getCounty = this.getCounty.bind(this);
+    this.getTown = this.getTown.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.handleGuess = this.handleGuess.bind(this);
+    console.log(this.state.borderLayer);
+  }
+
+  // function that moves the current marker to a new position
+  moveMarker() {
+    // use of object destructuring
+    {console.log('moved marker')}
+    const { lat, lng } = this.state.markerPosition;
+    this.setState({
+      markerPosition: {
+        lat: lat + 0.0001,
+        lng: lng + 0.0001, 
+      }
+    });
+  };
+
+  // fuction handles what happens when the start button is clicked
+  clickStart() {
+    console.log('clicked start');
+    // start the game
+    this.startGame();
+  }
+
+  // function that picks a random lat and long,
+  // centers the map on that spot, and zooms in
+  async startGame() {
+    const { borderLayer } = this.state;
+    console.log(borderLayer);
+    // there might be a better way to do this
+    const vtBounds = borderLayer.getBounds();
+    const xMax = vtBounds.getEast();
+    const xMin = vtBounds.getWest();
+    const yMax = vtBounds.getSouth();
+    const yMin = vtBounds.getNorth();
+
+    // find random lat and long 
+    // -> pull this out and generalize to find 
+    // random points in a given polygon/border?
+    let randomLatLon = () => {
+      // get random numbers
+      const lat = yMin + (Math.random() * (yMax - yMin));
+      const lon = xMin + (Math.random() * (xMax - xMin));
+      console.log(`${lat} ${lon}`);
+      // test if inside border
+      const layerLength = leafletPip.pointInLayer(
+                            [lon, lat], borderLayer, true
+                          ).length;
+      console.log({layerLength});
+      const result = layerLength ? "yes" : "no";
+      console.log({result});
+      
+      // if inside border, return the lat and lon as an object
+      if (result === "yes") {
+        return {lat:lat, lon: lon};
+      //else return randomLatLon()
+      } else {
+        return randomLatLon();
+      }
+      
+    }
+    //console.log(` random: ${JSON.stringify(randomLatLon())}`);
+    const { lat, lon } = randomLatLon();
+    const county = this.getCounty();
+    const town = await this.getTown(lat, lon);
+    console.log(`${town} ${county}`);
+
+    // set marker position to new randomLatLon
```

Comment:
You only explained the change in marker positions. Explain the change in the other state values here.